.github/workflows/go.yml

```
@@ -0,0 +1,23 @@
+name: Go
+
+on: [push]
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: github.com/kohofinancial/errors
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
```

This is an ancient version, and you can tell setup-go to use the Go version from the `go.mod` file:

```suggestion
        uses: actions/setup-go@v3.3.0
        with:
          go-version-file: go.mod
```

That's the path to the `go.mod` file (see [docs](https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file)), i.e., would have to change if you set a path in the checkout step.